ci/conan/all/conanfile.py

```diff
@@ -268,7 +268,9 @@ def validate(self):
             raise ConanInvalidConfiguration("arrow:parquet requires arrow:with_thrift")
 
     def build_requirements(self):
-        if Version(self.version) >= "13.0.0":
+        if Version(self.version) >= "20.0.0":
+            self.tool_requires("cmake/[>=3.25 <4]")
+        else:
```

Comment:
Could you keep `ci/conan/**` files untouched?

`ci/conan/**` files are synchronized by `ci/conan/merge_upstream.sh` from upstream. If we change `ci/conan/**` in apache/arrow, the merge script may report many conflicts.

Reply:
Is there a reason we haven't kept them in sync with upstream then?
https://github.com/conan-io/conan-center-index/blob/7e4cf19b663a2cfa7e9a57b8260ba003e046a46c/recipes/arrow/all/conanfile.py#L256-L260

Reply:
Because it was added after https://github.com/apache/arrow/pull/46758 .